components/Spotlight/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import useSWR from "swr";
+import Image from "next/image";
+
+export default function SpotlightPiece({ image, artist }) {
```

Comment:
müssen es in unserer Version dann umbenennen, nutzen den Komponentennamen "Spotlight"